frontend/src/app/diff-versions/diff-versions.component.html

```diff
@@ -0,0 +1,121 @@
+<div class="container top-buffer">
+  <div class="row top-buffer">
+    <div class="alert alert-danger" role="alert" *ngIf="error">
+      {{ error }}
+    </div>
+  </div>
+
+  <h4>View changes between specification versions</h4>
+
+  <div class="row top-buffer">
+    <div class="col">
+      <table class="table">
+        <thead class="thead">
+        <tr>
+          <th scope="col">Specification title</th>
+          <th scope="col">Version</th>
+          <th scope="col">+</th>
+          <th scope="col">-</th>
+        </tr>
+        </thead>
+        <ng-container *ngFor="let specification of (searchComplete ? service.specifications : [])">
+          <tbody *ngIf="service.specifications.length > 1">
+          <tr>
+            <td><a [routerLink]="['specifications/', service.id, specification.metadata.version]" routerLinkActive="active">{{ service.title }}</a></td>
+            <td>{{ specification.metadata.version }}</td>
+            <td>
+              <input type="radio" name="radiogroup-before" [value]="specification.metadata.version" (change)="radioButtonChange($event)" [(ngModel)]="radioSelectedBeforeVersion"/>
```

Comment:
I'm not sure if this too complicated. I always hate that config dialog where I have to choose two versions to compare. What do you think about automatically comparing it with the previous version?

Reply:
Just to clarify, are you proposing a) pre-setting the radio buttons to versions latest and (latest-1), b) removing the radio button selection entirely and only ever displaying the results of latest vs (latest-1), or c) only having one column of radio buttons, which displays the diff between version n and version (n-1)?

Reply:
My idea was b). However, that is just an idea I'd like you input on :)

Reply:
I think ultimately it's useful to have the flexibility of being about to choose any pair, for example to compare across major versions. (As a compromise, I could have multiple views as part of the "diff-versions" Angular page, where the user first sees the results of b), however can navigate to a tab with a more powerful but complex interface?)

Reply:
Ideally I wanted to UI element to be a vertical double-ended range slider, sort of like https://codepen.io/Gyuszko/pen/QrJoEz However I could not find a way to integrate this into an html table, as the table rows correspond to the slider options.